### DISCUSSION TITLE
Comma separated queries

### DISCUSSION THREAD

**Discussion by fustilio** [tim-hub/obsidian-bible-reference#173] [Ideas]
Thank you for your work,

I'm not sure if this is already available but I'd have expected that queries with comma separated verses from the same chapter would be selected and concatenated. If it is not already available, I'm keen to work on this feature! :-)

<img width="479" alt="image" src="https://github.com/tim-hub/obsidian-bible-reference/assets/23294131/71c53dd9-04a9-418d-9053-e1b827cd1457">

**Comment by tim-hub** [comment]
@fustilio It is not at current stage, that will be great if you want to contribute on this function.
